server/plugin/webhook.go

```diff
@@ -61,6 +62,13 @@ func ConvertPushEventRepositoryToRepository(pushRepo *github.PushEventRepository
 func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 	config := p.getConfiguration()
 
+	if config.EnableWebhookEventLogging {
+		parsedRequest, err := httputil.DumpRequest(r, true)
+		if err != nil {
+			p.API.LogWarn("Error while parsing Webhook Request")
```

Comment:
`return` here?

Reply:
didn't wanted to effect the existing flow , if some thing went wrong we wont be processing the event  so removed the return explicitly , let me know if you want me to add return

Reply:
`parsedRequest` is not a usable value if an error occurred, so the next line below that tries to use `parsedRequest` is invalid. We either need to return, or put the next line in an else block. I don't think we should expect an error to be returned from `DumpRequest` when it is a valid request from GitHub, so 1/5 let's return here:

```suggestion
			p.API.LogWarn("Error while parsing Webhook Request", "err", err.Error())
			return
```